src/main/java/com/kolya/gym/validation/CommonValidation.java

```diff
@@ -0,0 +1,10 @@
+package com.kolya.gym.validation;
+
+public class CommonValidation {
+    static public void nullValidation(Object object) throws IllegalArgumentException{
+        if (object==null){
+            throw new IllegalArgumentException("Input Data is null");
```

Comment:
Better to use `Optional` class instead of a separate class for null verification.

Reply:
I replaced all "nullValidation()" functions with "Optional.ofNullable().OrElseThrow()"